test/groovy/org/ods/orchestration/DeployStageSpec.groovy

```diff
@@ -66,8 +66,8 @@ class DeployStageSpec extends SpecHelper {
 
         then:
         1 * bitbucketService.getUrl() >> "https://bitbucket"
-        1 * project.targetClusterIsExternal >> false
-        1 * openshiftService.envExists(project.targetProject) >> true
+        1 * util.getInstallableRepos()
+        0 * util.verifyEnvLoginAndExistence(*_)
```

Comment:
Why in a test "deploy in Q with 3 installable repos (all included)" `util.verifyEnvLoginAndExistence` is not called? it should be.